src/bindings/python/valhalla/utils.py

```diff
@@ -0,0 +1,62 @@
+from typing import List, Tuple
```

Comment:
this is also new. it's basically copied code from https://github.com/hicsail/polyline. it's so small and will hardly ever change, I figured it's better to distribute it ourselves

Reply:
btw, I begun writing `TypedDictionary` definition for the valhalla data. Would that be of interest to ship with the bindings?

If so; as inline types or separate type definitions? (Not sure how old Python versions this project officially support.)